DRODLib/OrbUtil.cpp

```diff
@@ -122,8 +117,8 @@ void OrbUtil::MergeYellowDoorConnectionsInArea(
 
 	vector<COrbData*> orbData;
 
-	for (UINT y = wX; y <= endY; ++y) {
-		for (UINT x = wY; x <= endX; ++x) {
+	for (UINT x = wX; x <= endX; ++x) {
```

Comment:
I blame myself for not spotting this.
Your additional optimizations are great, though!  Thank you.